framework/src/org/checkerframework/common/value/qual/MinLen.java

```diff
@@ -1,18 +1,14 @@
-package org.checkerframework.checker.index.qual;
+package org.checkerframework.common.value.qual;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Target;
-import org.checkerframework.framework.qual.DefaultQualifierInHierarchy;
-import org.checkerframework.framework.qual.SubtypeOf;
 
 /**
  * The value of the annotated expression is a sequence containing at least the given number of
- * elements.
+ * elements. An alias for an {@link ArrayLenRange} annotation with only a {@code from} field.
  *
  * @checker_framework.manual #index-checker Index Checker
  */
-@DefaultQualifierInHierarchy
-@SubtypeOf({})
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
 public @interface MinLen {
```

Comment:
>I think @MinLen is preferable because we'd like to eventually expand it to also be useful for Strings and possibly lists in the future.

At that time `@ArrayLen` and `@ArrayLenRange` would have to be changed, too.